lib/pages/home.dart

```diff
@@ -86,7 +86,7 @@ class HomePage extends StatelessWidget {
           package: 'flutter_gallery_assets',
         ),
         assetDarkColor: const Color(0xFF253538),
-        studyRoute: RallyApp.loginRoute,
```

Comment:
It looks like the Shrine and Starter app are not using the new routes files you added yet

Reply:
Done.